Describe devmode for containers (#150)
anbox-cloud/discourse-docs: exp/containers.md

`deleted` | The container is deleted and will be removed from the AMS database soon.
`error` | An error occurred while processing the container. The container is stopped. Further information about the error can be viewed with `amc show <container id>`.

## Development mode

> AMS allows to start a container in a development mode. This will turn off certain features, which are active in a container otherwise. The mode is mainly useful for development of addons inside an Anbox container.

Reviewer: ```suggestion
AMS allows to start a container in development mode. This mode turns off some features that are usually active in a container. It is mainly useful when developing addons inside a container.
```
I was confused by the "Anbox container" here - but that's what we're talking about everywhere, right?

> When development mode is turned on, the container will not send status updates to AMS when the Anbox runtime is terminated. This allows you to restart the Anbox runtime inside the container providing an easy way to test [addons](https://discourse.ubuntu.com/t/addons/25293) or other extensions.

Reviewer: ```suggestion
When development mode is enabled, the container does not send status updates to AMS when the Anbox runtime is terminated. This allows you to restart the Anbox runtime inside the container, providing an easy way to test [addons](https://discourse.ubuntu.com/t/addons/25293) or other extensions.
```

> Inside an Anbox container development mode can be detected by looking at the `devmode` field in the `/var/lib/anbox/session.yaml` configuration file. If it is set to `true` then development mode is active, otherwise it is not.

Reviewer 2: Hmm, that's not very intuitive. Another more friendly way would be when people execute
``` 
amc show <container id> 
```

the output includes the devmode information.

Author: It's documented to allow code people write to detect inside a container if devmode is turned on or not. You're right, if you just want to see if it is enabled for a particular container `amc show` is the way to go.

Reviewer 2: Maybe it's worth mentioning that in the document if people want to know if a container is launched in devmode or not since that's the only way people could get this piece of information from the UI without entering the container and parsing the session.yaml file.

Reviewer: I agree, we should include it.
```suggestion
To check whether development mode is enabled, run `amc show <container_ID>` or look at the `/var/lib/anbox/session.yaml` file in the container. If the `devmode` field in the configuration file is set to `true`, development mode is active.
```
